scripts/create-workspace.js

```diff
@@ -0,0 +1,378 @@
+          }
+
+          if (nameExists(value)) {
+            return `Workspace name "${value}" is already taken. Please provide another name`;
+          }
+
+          return true;
+        }
+      },
+      {
+        type: 'text',
+        name: 'description',
+        message: 'Provide a one-line description of the workspace'
+      },
+      {
+        type: 'confirm',
+        name: 'isPublic',
+        message: 'Is it a public package?',
+        initial: true
+      },
+      {
+        type: 'confirm',
+        name: 'react',
+        message: 'Will the package use React?',
+        initial: true
+      },
+      {
+        type: 'autocompleteMultiselect',
+        name: 'dependants',
+        message: 'Will any of the packages in the monorepo depend on this one?',
+        choices: Array.from(workspacesMeta.values())
+          .sort((a, b) => a.name.localeCompare(b.name))
+          .map(({ name, location }) => ({ title: name, value: location })),
+        hint: '(select as many as you like, or none)',
+        instructions: `
+
+ · Use up and down arrows to navigate the list
+ · Use space to toggle the selection
+ · Type text to filter choices
+ · Enter to complete the answer
+`,
+        initial: []
+      },
+      {
+        type(prev) {
+          return prev.length > 0 ? 'select' : null;
+        },
+        name: 'depType',
+        message: 'What type of dependency is it?',
+        choices: [
+          { title: 'Production', value: 'dependencies' },
+          { title: 'Development', value: 'devDependencies' }
+        ]
+      }
+    ],
+    {
+      onCancel() {
+        canceled = true;
+      }
+    }
+  );
+
+  if (canceled) {
+    return;
+  }
+
+  console.log();
+
+  const pkgJson = {
+    name: dirToScopedPackageName(name),
+    ...(description && { description }),
+    author: {
+      name: 'MongoDB Inc',
+      email: 'compass@mongodb.com'
+    },
+    ...(isPublic ? { publishConfig: { access: 'public' } } : { private: true }),
+    bugs: {
+      url: 'https://jira.mongodb.org/projects/COMPASS/issues',
+      email: 'compass@mongodb.com'
+    },
+    homepage: 'https://github.com/mongodb-js/compass',
+    version: '0.1.0',
+    repository: {
+      type: 'git',
+      url: 'https://github.com/mongodb-js/compass.git'
+    },
+    files: ['dist'],
+    license: 'SSPL',
+    main: 'dist/index.js',
+    exports: {
+      require: './dist/index.js',
+      import: './dist/.esm-wrapper.mjs'
+    },
+    types: './dist/index.d.ts',
+    scripts: {
+      bootstrap: 'npm run compile',
+      prepublishOnly: 'npm run compile',
+      compile:
+        'tsc -p tsconfig.json && gen-esm-wrapper . ./dist/.esm-wrapper.mjs',
+      eslint: 'eslint',
+      prettier: 'prettier',
+      lint: 'npm run eslint . && npm run prettier -- --check .',
+      depcheck: 'depcheck',
+      check: 'npm run lint && npm run depcheck',
+      'check-ci': 'npm run check',
+      test: 'cross-env TS_NODE_FILES=true nyc mocha',
```

Comment:
was fighting with this a lot, can we have a different task for covered tests (`test-cov`)? And a `test-watch` with `--watch`, maybe only me, but personally I use that a ton and run tests each 3 seconds, i would love to have a way for that to be really fast.

Would you be ok with including the changes i've done here: (also fixes nyc reporting coverage in test files)

https://github.com/mongodb-js/compass/pull/2412/files#diff-a557af5d61219f0b3f0b1fc6810efa3a9e0889fc54005cd248908a50cdf1d612L40

Reply:
Totally, let's include this stuff in the template! Then following your suggestion we will scripts like this:

- `test` -> `mocha`
- `test-cov` -> `cross-env TS_NODE_FILES=true nyc -x "**/*.spec.*" npm run test`
- `test-watch` -> `npm run test -- --watch`
- `test-ci` -> `npm run test-cov`

Makes sense?

Reply:
Yep!

Reply:
Done in 34089b9